DesktopModules/Vanjaro/UXManager/Extensions/Block/Register/Views/Setting/App_LocalResources/Register.resx

```diff
@@ -135,6 +135,9 @@
   <data name="IsPageAdminAlert.Text" xml:space="preserve">
     <value>&lt;strong&gt;&lt;em class="fa fa-info-circle"&gt;&lt;/em&gt; Info&lt;/strong&gt;: This is a theme block. Only administrators can make changes to Site Settings. You may customize this block only.</value>
   </data>
+  <data name="ReCaptcha_Error.Text" xml:space="preserve">
```

Comment:
Can we get the recaptcha error message from a global shared file. I don't want it repeated. See the message I added in Login.resx and use that in a Shared.resx

Reply:
changes done